diff --git a/node_modules/expo-speech-recognition/src/ExpoWebSpeechRecognition.web.ts b/node_modules/expo-speech-recognition/src/ExpoWebSpeechRecognition.web.ts
index dd03fc5..a3bfe7c 100644
--- a/node_modules/expo-speech-recognition/src/ExpoWebSpeechRecognition.web.ts
+++ b/node_modules/expo-speech-recognition/src/ExpoWebSpeechRecognition.web.ts
@@ -2,7 +2,7 @@ let browserSpeechRecognition: typeof SpeechRecognition | null = null;
 let browserSpeechGrammarList: typeof SpeechGrammarList | null = null;
 let browserSpeechRecognitionEvent: typeof SpeechRecognitionEvent | null = null;
 
-if (typeof webkitSpeechRecognition !== "undefined") {
+if (typeof webkitSpeechRecognition !== "undefined" && typeof webkitSpeechGrammarList !== "undefined" && typeof webkitSpeechRecognitionEvent !== "undefined") {
   browserSpeechRecognition = webkitSpeechRecognition;
   browserSpeechGrammarList = webkitSpeechGrammarList;
   browserSpeechRecognitionEvent = webkitSpeechRecognitionEvent;
