diff --git a/node_modules/expo-speech-recognition/build/ExpoWebSpeechRecognition.web.js b/node_modules/expo-speech-recognition/build/ExpoWebSpeechRecognition.web.js
index e8b8547..59a95f4 100644
--- a/node_modules/expo-speech-recognition/build/ExpoWebSpeechRecognition.web.js
+++ b/node_modules/expo-speech-recognition/build/ExpoWebSpeechRecognition.web.js
@@ -1,26 +1,25 @@
 let browserSpeechRecognition = null;
 let browserSpeechGrammarList = null;
 let browserSpeechRecognitionEvent = null;
-if (typeof webkitSpeechRecognition !== "undefined") {
+
+if (
+    typeof webkitSpeechRecognition !== "undefined" &&
+    typeof webkitSpeechGrammarList !== "undefined" &&
+    typeof webkitSpeechRecognitionEvent !== "undefined"
+) {
     browserSpeechRecognition = webkitSpeechRecognition;
-    browserSpeechGrammarList =
-        typeof webkitSpeechGrammarList !== "undefined"
-            ? webkitSpeechGrammarList
-            : null;
-    browserSpeechRecognitionEvent =
-        typeof webkitSpeechRecognitionEvent !== "undefined"
-            ? webkitSpeechRecognitionEvent
-            : null;
-}
-else if (typeof SpeechRecognition !== "undefined") {
+    browserSpeechGrammarList = webkitSpeechGrammarList;
+    browserSpeechRecognitionEvent = webkitSpeechRecognitionEvent;
+} else if (
+    typeof SpeechRecognition !== "undefined" &&
+    typeof SpeechGrammarList !== "undefined" &&
+    typeof SpeechRecognitionEvent !== "undefined"
+) {
     browserSpeechRecognition = SpeechRecognition;
-    browserSpeechGrammarList =
-        typeof SpeechGrammarList !== "undefined" ? SpeechGrammarList : null;
-    browserSpeechRecognitionEvent =
-        typeof SpeechRecognitionEvent !== "undefined"
-            ? SpeechRecognitionEvent
-            : null;
+    browserSpeechGrammarList = SpeechGrammarList;
+    browserSpeechRecognitionEvent = SpeechRecognitionEvent;
 }
+
 export const ExpoWebSpeechRecognition = browserSpeechRecognition;
 export const ExpoWebSpeechGrammarList = browserSpeechGrammarList;
 export const ExpoWebSpeechRecognitionEvent = browserSpeechRecognitionEvent;
